g11n-ws/modules/md-service-i18n-l3/src/main/java/com/vmware/vip/core/messages/service/multcomponent/MultComponentService.java

```diff
@@ -90,5 +85,20 @@ private TranslationDTO getTranslation(TranslationDTO translationDTO)
 		translationDTO.setBundles(ja);
 		return translationDTO;
 	}
+
+	@Override
+	public List<ResultMessageChannel> getTranslationChannels(String productName, String version,
+			List<String> components, List<String> locales) throws L3APIException {
+		
+		try {
+			return componentChannelDao.getTransReadableByteChannels(productName, version, components, locales);
+		} catch (DataException e) {
+			LOGGER.error(e.getMessage(), e);
+			throw new L3APIException(
+					"Faild to get translation from data for " + productName + ConstantsChar.BACKSLASH + version, e);
```

Comment:
It would be better to add components and locales in the log

Reply:
The log is the response massage can't change

Reply:
the log message is response message, it can't change